### PR TITLE
Update laravel guide

### PR DESCRIPTION
**Pull request by irsyadadl**
Laravel now use laravel mix v6, so I make a change for the version to use latest instead of compat-7, and I also add purge to include "Laravel Views Class Component".

### CODE REVIEW

File: src/pages/docs/guides/laravel.mdx [hunk 1]
```diff
@@ -27,6 +27,7 @@ types:
   - Blade templates
   - JavaScript components
 purge: 
+  - ./app/**/*.php
```
Comment (irsyadadl) [inline src/pages/docs/guides/laravel.mdx:30]: Add purge to "Laravel Component Class"

File: src/pages/docs/guides/laravel.mdx [hunk 2]
```diff
@@ -41,6 +42,7 @@ In your `webpack.mix.js`, add `tailwindcss` as a PostCSS plugin:
   mix.js("resources/js/app.js", "public/js")
     .postCss("resources/css/app.css", "public/css", [
 +     require("tailwindcss"),
++     require("autoprefixer"),
```
Comment (adamwathan) [inline src/pages/docs/guides/laravel.mdx:45]: Hey thanks for the PR! Laravel Mix adds autoprefixer automatically at least historically, did this change for v6?

Comment (irsyadadl) [reply src/pages/docs/guides/laravel.mdx:45]: Laravel breeze now use mix v6, I see the webpack.mix.js include it.

**Comment by adamwathan** [comment]
Thanks!

**Pull request merged**